### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSettings_ColorContrast:Color contrast ratio is less than 4.5:1 for the placeholder and the list items present on the screen.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52481]
**Environment Details:**
VSCode Version : 1.23.1
OS Version : Windows 10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1.Launch VS Code.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown
3. Open Color Contrast Analyzer.

**Actual**
Color contrast ratio is less than 4.5:1 for the placeholder and the list items present on the screen

**Expected:**
Color contrast ratio should be more  than 4.5:1 for the placeholder and the list items present on the screen

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The End user will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Attachment For Reference:**

![usersettings](https://user-images.githubusercontent.com/39204909/41661705-ecb58f26-74bc-11e8-8f32-cd5765f1fe34.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
Duplicate #52593
